packages/testing/src/api/index.ts

```diff
@@ -14,18 +14,23 @@
  * limitations under the License.
  */
 
-import { firebase } from '@firebase/app';
+import * as firebase from 'firebase';
 import * as request from 'request';
-import { FirebaseApp, FirebaseOptions } from '@firebase/app-types';
 import { base64 } from '@firebase/util';
 import { setLogLevel, LogLevel } from '@firebase/logger';
 import * as grpc from 'grpc';
+import { resolve } from 'path';
 import * as fs from 'fs';
 
-const PROTOS = grpc.load({
-  root: __dirname + '/../protos',
+const PROTO_ROOT = {
+  root: resolve(
+    __dirname,
+    process.env.FIRESTORE_EMULATOR_PROTO_ROOT || '../protos'
```

Comment:
the protos are in the `src` directory during tests but in a different places when we package and distribute them, so this is a workaround. This is how `@firebase/firestore` handles this issue.